templates/parachain/runtime/src/configs/mod.rs

```diff
@@ -250,6 +250,22 @@ impl pallet_message_queue::Config for Runtime {
 
 impl cumulus_pallet_aura_ext::Config for Runtime {}
 
+parameter_types! {
+	/// The asset ID for the asset that we use to pay for message delivery fees.
+	pub FeeAssetId: AssetId = AssetId(xcm_config::RelayLocation::get());
```

Comment:
Interesting. I thought the template had you use your own token but it seems like it uses the relay token? That makes me think it was not thought through. Since if you wnated to use for example DOT you'd need a first step where you cross-chain transfer it in. Probably not everyone wants to have a DOT-powered chain either. We should change this but it's out of scope for this PR